### PR TITLE
LPS-132250 Provide on-demand admin access to virtual instances

### PR DESCRIPTION
**Pull request by pei-jung**

### CODE REVIEW

**Comment by Sharryxue** [comment]
No specific issue found in ci:test:relevant suite.

File: modules/apps/portal-instances/portal-instances-on-demand-admin/src/main/java/com/liferay/portal/instances/on/demand/admin/internal/messaging/OnDemandAdminCleanerMessageListener.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+	}
+
+	@Modified
+	protected void modified(Map<String, Object> properties) {
+		deactivate();
+
+		activate(properties);
+	}
+
+	private OnDemandAdminConfiguration _onDemandAdminConfiguration;
```
Comment (drewbrokke) [inline modules/apps/portal-instances/portal-instances-on-demand-admin/src/main/java/com/liferay/portal/instances/on/demand/admin/internal/messaging/OnDemandAdminCleanerMessageListener.java:91]: I believe this field needs to be `volatile`.

**Review by drewbrokke — CHANGES_REQUESTED** [review]
Just one inline comment. Otherwise I think this is ready to go.

**Comment by pei-jung** [comment]
ci:forward

**Pull request closed without merging**